src/services/milestones/milestones.hooks.js

```diff
@@ -201,6 +203,60 @@ const sendNotification = () => context => {
   }
 };
 
+/***
+ * This function checks that the maxAmount in the milestone is based on the correct conversion rate of the milestone date
+ **/
+const checkEthConversion = () => (context) => {
+  // abort check for internal calls
+  if (!context.params.provider) return context;
+
+  const { data, app } = context;
+  let maxAmount = 0;
+  const items = data.items;
+
+  const calculateCorrectEther = (conversionRate, fiatAmount, etherToCheck, selectedFiatType) => {
+    logger.info('calculating correct ether conversion', conversionRate.rates[selectedFiatType], fiatAmount, etherToCheck);
+    // calculate the converion of the item, make sure that fiat-eth is correct
+    const rate = conversionRate.rates[selectedFiatType];
+    const ether = utils.toWei((fiatAmount / rate).toFixed(18));
```

Comment:
I'm not sure this will always work...

If I do `(1/5).toFixed(18)` it returns `'0.200000000000000011'`. The `11` I think will cause issues.

Might be safer to use BN.js which is accessible via `uitls.BN` to do this calculation & check

Reply:
ok did't know there was BN already available, will change